tests/hpc/spack_slave.pm

```diff
@@ -44,9 +45,10 @@ sub post_fail_hook ($self) {
     # Inside compiled modules comes as <module>-<version>-<hash>
     # Each module includes build logs under <module>-<version>-<hash>/.spack
     my $compiler_ver = script_output("gcc --version | grep -E '\\b[0-9]+\.[0-9]+\.[0-9]+\$' | awk '{print \$4}'");
+    record_info "postfail", "$compiler_ver";
     my $arch = get_var('ARCH');
     my $node = script_output('hostname');
-    tar_and_upload_log("/opt/spack/linux-sle_hpc15-$arch/gcc-$compiler_ver", "/tmp/spack_$node.tar.bz2");
+    #tar_and_upload_log("/opt/spack/linux-sle_hpc15-$arch/gcc-$compiler_ver", "/tmp/spack_$node.tar.bz2");
```

Comment:
either `#` or whole line  should be removed

Reply:
i had to squash and rebase which asked me to fix some conflicts. i missed that

Reply:
fixed and ready for review.